agent/requirements.txt

```diff
@@ -55,3 +55,4 @@ werkzeug==0.16.1
 result==0.5.0
 wtforms-json==0.3.3
 alembic==1.4.2
+regex==2020.11.13
```

Comment:
because re doesn't support recursion, I use it here to extract function arguments https://github.com/anodot/daria/blob/58389798ad6df28628be45d0d71b0124a65597ff/agent/src/agent/pipeline/config/expression_parser/condition.py#L170